tests/ui/conftest.py

```diff
@@ -14,31 +18,120 @@ def capabilities(capabilities):
     capabilities['marionette'] = True
     return capabilities
 
+
 @pytest.fixture
 def fxa_account(base_url):
-    url = DEV_URL if 'dev' in base_url else PROD_URL
+    url = DEV_URL if 'dev' or 'localhost' in base_url else PROD_URL
     return FxATestAccount(url)
 
 
-@pytest.fixture(scope='session')
+@pytest.fixture
 def jwt_issuer(base_url, variables):
     try:
-        hostname = [urlparse.urlsplit(base_url).hostname]
+        hostname = urlparse.urlsplit(base_url).hostname
         return variables['api'][hostname]['jwt_issuer']
     except KeyError:
         return os.getenv('JWT_ISSUER')
 
 
-@pytest.fixture(scope='session')
+@pytest.fixture
 def jwt_secret(base_url, variables):
     try:
-        hostname = [urlparse.urlsplit(base_url).hostname]
+        hostname = urlparse.urlsplit(base_url).hostname
         return variables['api'][hostname]['jwt_secret']
     except KeyError:
         return os.getenv('JWT_SECRET')
 
 
 @pytest.fixture
+def initial_data(transactional_db, live_server):
+    call_command('generate_addons', 10, app='firefox')
+
+
+@pytest.fixture
+def create_superuser(transactional_db, live_server, base_url, tmpdir):
+    create_switch('super-create-accounts')
+    call_command('loaddata', 'initial.json')
+
+    call_command(
+        'createsuperuser',
+        interactive=False,
+        username='uitest',
+        email='uitester@mozilla.org',
+        add_to_supercreate_group=True,
+        save_api_credentials=str(tmpdir.join('variables.json')),
+        hostname=urlparse.urlsplit(base_url).hostname
+    )
+
+
+@pytest.fixture
+def force_user_login():
+    from olympia.users.models import UserProfile
+    user = UserProfile.objects.get(username='uitest')
+    return user
```

Comment:
We can just `return UserProfile.objects.get(username='uitest')` without the need for the `user` variable.